cockroachcloud/private-clusters.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Create private CockroachDB Dedicated clusters (Preview)
+summary: Learn how to create CockroachDB Dedicated clusters with no node-level public IP addresses.
+toc: true
+docs_area: manage.security
+---
+
+{% include feature-phases/preview-opt-in.md %}
+
+Limiting access to a CockroachDB cluster's nodes over the public internet is an important security practice and is also a compliance requirement for many organizations. {{ site.data.products.dedicated }} private clusters allow organizations to meet this objective.
+
+By default, {{ site.data.products.db }} has safeguards in place to protect cluster's data from the public internet. Ingress traffic to a cluster is routed through a load balancer, and it's possible to restrict inbound connections using a combination of [IP allowlisting](/docs/cockroachcloud/network-authorization.html#ip-allowlisting) and [AWS PrivateLink](/docs/cockroachcloud/network-authorization.html#aws-privatelink) or [GCP VPC peering](/docs/cockroachcloud/network-authorization.html#vpc-peering). However, data egress operations such as exports, backups, and Change Data Capture (CDC) use public subnets.
+
+On the other hand, a private {{ site.data.products.dedicated }} cluster's nodes have no public IP addresses, and egress traffic moves over private subnets through a highly-available NAT gateway that is unique to the cluster. This page explains what happens when you create a private cluster. Existing clusters that are not created as private clusters can't be switched in place, unless you choose to perform full data migration to a new private cluster.
```

Comment:
> Existing clusters that are not created as private clusters can't be switched in place, unless you choose to perform full data migration to a new private cluster.

This line feels tacked on here; perhaps either add a line break or move it below the following callout.

Also (speaking as migration docs writer), what is meant by "full data migration to a new private cluster"? Is that something that we enable, and if so, can we provide a link or hint on how this is accomplished on Dedicated?

Reply:
I moved this to a new section at the bottom of the page, and added a link to the Migration page.